packages/cli/src/index.ts

```diff
@@ -0,0 +1,51 @@
+import { command, run, string } from "@drizzle-team/brocli";
+
+import { hashPasswordAsync } from "@homarr/auth/security";
+import { and, db, eq } from "@homarr/db";
+import { sessions, users } from "@homarr/db/schema/sqlite";
+
+const resetPassword = command({
+  name: "reset-password",
+  desc: "Reset password for a user",
+  options: {
+    username: string("username").required().alias("u").desc("Name of the user"),
+  },
+  // eslint-disable-next-line no-restricted-syntax
+  handler: async (options) => {
+    if (!process.env.AUTH_PROVIDERS?.toLowerCase().includes("credentials")) {
+      console.error("Credentials provider is not enabled");
+      process.exit(1);
+    }
+
+    const user = await db.query.users.findFirst({
+      where: and(eq(users.name, options.username), eq(users.provider, "credentials")),
+    });
+
+    if (!user?.salt) {
+      console.error(`User ${options.username} not found`);
+      process.exit(1);
+    }
+
+    const newPassword = crypto.randomUUID();
```

Comment:
Is UUID a good idea for a password? I'd prefer a secure random generator from the crypto module instead